src/module/actor/character/document.ts

```diff
@@ -654,7 +648,9 @@ class CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
         // Armor Class
         const armorStatistic = this.createArmorStatistic();
         this.armorClass = armorStatistic.dc;
-        systemData.attributes.ac = armorStatistic.getTraceData();
+        systemData.attributes.ac = fu.mergeObject(armorStatistic.getTraceData(), {
+            attribute: armorStatistic.attribute ?? "dex",
+        });
```

Comment:
only just starting to look at this, but is there any reason getTraceData() can't take care of this attribute property?

Reply:
I didn't write the method. It doesn't spit out rank or attribute.